regexes/device/mobiles.yml

```
@@ -24160,7 +24160,7 @@ Beelink:
 
 # MAXVI
 MAXVI:
-  regex: 'MAXVI[ _](?:MS401)(?:[);/ ]|$)|(?:\w+)?_MAXVI_\w+|MS502[_ ]Orion|MS531'
+  regex: 'MAXVI[ _](?:MS401)(?:[);/ ]|$)|.*_?MAXVI?_\w|MS502[_ ]Orion|MS531'
```

That doesn't seem to be the same as before either:
```suggestion
  regex: 'MAXVI[ _](?:MS401)(?:[);/ ]|$)|\w*_?MAXVI?_\w+|MS502[_ ]Orion|MS531'
```